docs/1-Getting-Started/2-Starting-new-project.md

```diff
@@ -18,164 +12,164 @@ Also, make sure you're running the latest version of **kool**. Run the following
 $ kool self-update
 ```
 
-### 1. Run `kool create`
+### 1. Create a New Project
 
-Use the `kool create <preset> <project-name>` command to create your new project.
+Create a working directory for your new project, and move into it.
 
 ```bash
-$ kool create laravel my-project
+$ mkdir my-app
+$ cd my-app
 ```
 
-Under the hood, this command will run `composer create-project --no-install --no-scripts --prefer-dist laravel/laravel <project-name>` (using a customized **kool** Docker image: <a href="https://github.com/kool-dev/docker-php" target="_blank">kooldev/php:7.4</a>).
+### 2. Add docker-compose.yml
 
-After installing Laravel, `kool create` automatically runs the `kool preset laravel` command, which helps you easily set up the initial tech stack for your project using an interactive wizard.
+Create a **docker-compose.yml** file in the root of your working directory, and then copy/paste one of the YAML templates provided below into **docker-compose.yml**.
 
 ```bash
-Preset laravel is initializing!
-
-? What app service do you want to use [Use arrows to move, type to filter]
-  PHP 7.4
-> PHP 8.0
-
-? What database service do you want to use [Use arrows to move, type to filter]
-> MySQL 8.0
-  MySQL 5.7
-  PostgreSQL 13.0
-  none
-
-? What cache service do you want to use [Use arrows to move, type to filter]
-> Redis 6.0
-  Memcached 1.6
-  none
-
-? What javascript package manager do you want to use [Use arrows to move, type to filter]
-  npm
-> yarn
-
-? What composer version do you want to use [Use arrows to move, type to filter]
-  1.x
-> 2.x
-
-$ Preset laravel initialized!
+$ touch docker-compose.yml
 ```
 
-Now, move into your new project.
+Here's a simple, generic setup for **docker-compose.yml**:
 
-```bash
-$ cd my-project
+```yaml
+version: "3.8"  # optional since v1.27.0
+services:
+  app:
+    image: kooldev/php:8.0-nginx
+    ports:
+      - "80:80"
+    volumes:
+      - .:/app:delegated
 ```
 
-The `kool preset` command auto-generated the following configuration files and added them to your project, which you can modify and extend.
+Here's a more extensible, **Kool-optimized** setup for **docker-compose.yml** into which you can easily add additional services (i.e. database, cache, etc):
```

Comment:
As said above, I would like making this only a call instead of actually providing right here the preset version as well.